extensions.json

```diff
@@ -1476,7 +1476,8 @@
     },
     {
       "id": "Swellaby.vscode-rust-test-adapter",
-      "repository": "https://github.com/swellaby/vscode-rust-test-adapter"
+      "repository": "https://github.com/swellaby/vscode-rust-test-adapter",
+      "prepublish": "npm update vscode && npm install && npm run build"
```

Comment:
Ah, unfortunately, I don't think a `prepublish` step will fix the failing `npm install`, because events actually happen in this order:

1. recursively clone the extension repository
2. if there is a `yarn.lock` file, run `yarn install`, otherwise run `npm install`
3. next run any `prepublish` step

See also:

https://github.com/open-vsx/publish-extensions/blob/eee429cb14e55f23132a1e6c8192da6ce876271e/publish-extensions.js#L113-L124